docs/howto/operate/new-cluster/aws.md

```diff
@@ -101,6 +105,8 @@ you at least one core node running.
 
 ### Grant access to other users
 
+<!-- Is this section still relevant if the account is managed by SSO? -->
+
```

Comment:
@damianavila @yuvipanda Your input would be appreciated here

Reply:
@sgibson91 unfortunately yes! However, the need for this is a lot lesser now thanks to your `use-cluster-credentials`!